client/dashboard/components/date-range-picker/index.tsx

```diff
@@ -111,32 +107,33 @@ function DateRangePickerInner( {
 	showTwoMonths: boolean;
 	start: Date;
 	end: Date;
-	timezoneString?: string;
-	gmtOffset?: number;
 	onChange: ( next: { start: Date; end: Date } ) => void;
 	onClose: () => void;
+	timezoneString?: string;
+	gmtOffset?: number;
 	mobileLabelId: string;
 	desktopLabelId: string;
 	disableFuture: boolean;
 } ) {
 	const [ fromDraft, setFromDraft ] = useState< Date | undefined >( () => start );
 	const [ toDraft, setToDraft ] = useState< Date | undefined >( () => end );
-	const [ fromStr, setFromStr ] = useState( () => formatYmd( start, timezoneString, gmtOffset ) );
-	const [ toStr, setToStr ] = useState( () => formatYmd( end, timezoneString, gmtOffset ) );
+	// Use UTC formatting without timezone conversion for consistency
+	const [ fromStr, setFromStr ] = useState( () => format( start, 'yyyy-MM-dd' ) );
+	const [ toStr, setToStr ] = useState( () => format( end, 'yyyy-MM-dd' ) );
 	// Tracks the keyboard-focused preset in the listbox (roving focus), not the selected preset.
 	const [ compositeActiveId, setCompositeActiveId ] = useState< string | null >( null );
 
 	const today = useMemo( () => {
```

Comment:
I thought "today" would be according to the site, not according to UTC or the user's current timezone 🤔